docs/preview/connected-registry/quickstart-deploy-connected-registry-kubernetes.md

```diff
@@ -0,0 +1,308 @@
+
+## Supported Kubernetes distributions
+
+## Prerequisites
+
+- [Install or upgrade Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) to version >= 2.30.0
+  - If you're using a local install, sign in with Azure CLI by using the [az login](https://docs.microsoft.com/en-us/cli/azure/reference-index#az_login) command. To finish the authentication process, follow the steps displayed in your terminal. See [Sign in with Azure CLI](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli) for additional sign-in options.
+- The Azure CLI commands in this article are formatted for the Bash shell. If you're using a different shell like PowerShell or Command Prompt, you may need to adjust line continuation characters or variable assignment lines accordingly. This article uses variables to minimize the amount of command editing required.
+- A connected registry resource in Azure as described in the [Create connected registry using the CLI](https://docs.microsoft.com/en-us/azure/container-registry/quickstart-connected-registry-cli) quickstart guide. Both, `ReadWrite` and `ReadOnly` modes will work for this scenario.
+- An up-and-running Kubernetes cluster. If you don't have one, you can create a cluster using one of these options:
+    - [Kubernetes in Docker](https://kind.sigs.k8s.io/)
+    - [K3s: Lightweight Kubernetes](https://rancher.com/docs/k3s/latest/quick-start/) cluster.
+    - Self-managed Kubernetes cluster using [Cluster API](https://cluster-api.sigs.k8s.io/user/quick-start.html)
+    - An [Azure Kubernetes Service](https://docs.microsoft.com/azure/aks/kubernetes-walkthrough) cluster
+- [Helm 3](https://helm.sh/docs/intro/install/) installed.
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) installed.
+- A `kubeconfig` file and context pointing to your cluster.
+
+## Node Setup Requirements
+
+The helm chart installs Kubernetes resources used to run a connected registry on your Kubernetes cluster. The connected registry runs as a singleton pod on one node of the cluster. In order to pull artifacts from the running connected registry, the user is responsible for configuring each node of the cluster that will pull from this connected registry. To pull from the connected registry over HTTPS, the user is responsible for configuring SSL certs on each node that will pull from the connected registry. To pull from the connected registry of HTTP, the user must update their container runtime settings to recognize the connected registry as "insecure". See more information below.
+
+> [!WARNING]
+> Pulling from and pushing to the connected registry over HTTP is not secure. It is recommended to setup TLS certificates.
+
+## Fetch the Connected Registry Helm Chart
```

Comment:
I assume all those steps are executed on the client machine...

Reply:
Yes, will clarify this step.

Reply:
Please see updated comment